podman_compose.py

```diff
@@ -928,7 +928,15 @@ def volume_ls(self, proj=None):
         volumes = output.splitlines()
         return volumes
 
-def normalize_service(service):
+def normalize_service(service, sub_dir=''):
+    if sub_dir and 'build' in service and 'context' in service['build']:
```

Comment:
does not seem to handle the case were context is passed as string build like this

```yaml
services:
    web1:
      build: ./context
```

Reply:
I will check it